string base_frame
---
bool trajectoryAvailable
nav_msgs/Path trajectory